dags/app/cloudflare_bot/dag.py

```diff
@@ -0,0 +1,74 @@
+"""
+Create bot network traffic
+"""
+import logging
+from datetime import datetime, timedelta
+
+import requests
+from airflow.sdk import Variable, dag, task
+
+DEFAULT_ARGS = {
+    "owner": "Henry Lee",
+    "depends_on_past": False,
+    "start_date": datetime(2025, 1, 1),
+    "retries": 2,
+    "retry_delay": timedelta(minutes=5),
+}
+
+logger = logging.getLogger(__name__)
+
+
+@dag(
+    default_args=DEFAULT_ARGS,
+    schedule="@hourly",
+    max_active_runs=1,
+    catchup=False,
+)
+def PYCONTW_ETL_BOT_v1():
+
+    @task
+    def GET_TOP_WEBSITES() -> list[str]:
```

Comment:
probably a good idea to start using lower case 
```suggestion
    def get_top_websties() -> list[str]:
```